src/main/java/com/alauda/jenkins/plugins/DevopsClientTools.java

```diff
@@ -50,7 +50,7 @@ public void buildEnvVars(EnvVars env) {
     }
 
     @Extension
-    @Symbol("oc")
+    @Symbol("alaudaClient")
```

Comment:
> @Symbol("alaudaClient")

同上，@Symbol("alaudaDevOpsClient")
```suggestion
    @Symbol("alaudaClient")
```